apps/lib/app_provider.c

```diff
@@ -64,6 +67,9 @@ static int opt_provider_path(const char *path)
 
 int opt_provider(int opt)
 {
+    const unsigned char given = provider_option_given;
+
+    provider_option_given = 0;
```

Comment:
Shouldn't this be `provider_option_given = 1;` ?

Reply:
Yeah :(